src/geovista/examples/vectors.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""Importable and runnable geovista example.
+
+Notes
+-----
+.. versionadded:: 0.4.0
+
+"""
+from __future__ import annotations
+
+import numpy as np
+import pyvista as pv
+
+import geovista
+import geovista as gv
+
+
+def main() -> None:
+    """Create vectors plotting inspired by cartopy."""
+    # make cool swirly pattern
+    sphere = pv.Sphere(gv.common.RADIUS)
+    vectors = np.vstack(
+        (
+            -sphere.points[:, 1],
+            sphere.points[:, 0],
+            sphere.points[:, 2] * 0.0,
+        )
+    ).T
+    sphere["vectors"] = vectors * 0.1
+    sphere.set_active_vectors("vectors")
+
+    # plot mesh
+    plotter = gv.GeoPlotter()
+    plotter.set_background("gray")
```

Comment:
@tkoyama010 You can remove the `plotter.set_background`, if you agree to adopt the `geovista.theme` above :+1:

Reply:
```suggestion
```